cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh

```diff
@@ -37,6 +38,23 @@ function deploy_sriov_operator {
   popd
 }
 
+function deploy_network_resource_injector {
+  WEBHOOK_PATH=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/network-resources-injector-${INJECTOR_GIT_HASH}
+  if [[ ! -d $WEBHOOK_PATH ]]; then
+    curl -L https://github.com/intel/network-resources-injector/archive/${INJECTOR_GIT_HASH}/network-resources-injector.tar.gz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/
+  fi
+
+  pushd $WEBHOOK_PATH
+    make image
+    docker tag network-resources-injector localhost:5000/network-resources-injector
+    sed -i 's/network-resources-injector:latest/registry:5000\/network-resources-injector:latest/g' ./deployments/server.yaml
+    docker push localhost:5000/network-resources-injector
+    _kubectl apply -f ./deployments/auth.yaml
+    _kubectl apply -f ./deployments/server.yaml
+  popd
```

Comment:
Why `pushd` and `popd`? this is enclosed in the function scope.

Reply:
Not sure I understood your comment and the relation between pushd and the scope of the function. 
I pushd in the hook checkout so I can run make and apply the files locally, as per the hook installation setup.
Another approach could have been to pass the full path of the makefile and of the manifests. Do you think it's better that way?